core/geometry/src/test/geometry3d/Angle.test.ts

```diff
@@ -129,7 +130,7 @@ describe("Angle.AlmostEqual", () => {
     ck.testFalse(Angle.createRadians(1.0e-20).isExactZero);
 
     ck.checkpoint("End Angle.AlmostEqual");
-    expect(ck.getNumErrors()).equals(0);
+    expect(ck.getNumErrors()).toEqual(0);
```

Comment:
Are these equals -> toEqual changes still needed?

Reply:
not necessarily, functionally both behave the same, but I was going to leave it in there as an example of switching over from chai to vitest's testing syntax. chai, out of the box, doesn't support asynchronous assertions like how vitest does. The author of vitest [recommends](https://github.com/vitest-dev/vitest/issues/2843#issuecomment-1424352509) to not mix and match, but if we want to continue using the assertions API, we can continue importing that from within vitest. I can remove it this, to avoid inconsistency

Reply:
I guess we could use some best practices for vitest going forward. What is the difference between `expect(1).equals(0)` and `expect(1).toEqual(0)`?

Reply:
For primitive types, they are the exact same, but objects are more complex. In vitest, when comparing the expected value to the actual value, you can choose between toBe() or toEqual(). The former is strict equality (think ===). The latter **deeply**, recursively checks the equality of all property fields, not the object's identity. [Reference](https://stackoverflow.com/questions/45195025/what-is-the-difference-between-tobe-and-toequal-in-jest). Chai's .[equals](https://www.chaijs.com/api/bdd/#method_equal)() (or `eq`) would check an object is strictly equal, while for deeply equals you would use `eql`. Unfamiliar people would not be able to tell as easily in chai, that `equal` and `eql` behave differently, whereas vitest tries to be more explicit in their method naming.

Reply:
Ah, I see. Nice explanation! All of the occurrences of ".equals" in core-geometry operate on numbers or booleans. So to stick with vitest syntax, it would be more natural to switch them to `.toBe`. If you want to do this, feel free. Otherwise, we'll do it.

Reply:
Yes, toBe would be appropriate in those cases. I'll defer to you to update the syntax, I'll be giving a demo internally next week on some things to catch when refactoring

Reply:
David can we do that in a separate follow up pr then?

Reply:
Looking forward to your demo!